packages/smooth_app/lib/pages/product/common/country_wiki_links.dart

```diff
@@ -0,0 +1,25 @@
+class CountryWikiLinks {
```

Comment:
Will do. Also why am i adding the TODO saying the code will be moved to openfoodfacts-dart?

Reply:
> Also why am i adding the TODO saying the code will be moved to openfoodfacts-dart?

Because part of the code (in our case, the map) will eventually be moved to openfoodfacts-dart.
When we code for Smoothie, we sometimes code things that can be reused by other openfoodfacts-dart users.

Reply:
Okay